package.json

```diff
@@ -138,7 +138,7 @@
     "tailwindcss": "1.1.3",
     "taskr": "1.1.0",
     "tree-kill": "1.2.2",
-    "typescript": "4.3.4",
+    "typescript": "4.4.2",
     "wait-port": "0.2.2",
```

Comment:
4.4.3 is available, let’s use the latest 🙂

Reply:
:+1: